src/app/auth/UserService.ts

```diff
@@ -78,6 +78,19 @@ class UserService {
   public async findOneWithProjection(q, projection) {
     return await User.findOne(q, projection);
   }
+
+  public async updateProfile(address: { city: string; street: string; street_nr: string; plz: number },
```

Comment:
Please put each parameter in a new line if they are too long and create types instead of putting composed types inline